NORDIC Embedded Software/nrf52 Patient Monitoring Device/pca10040/s132/arm5_no_packs/_build/acc_file.d

```diff
@@ -1,7 +1,6 @@
 .\_build\acc_file.o: ..\..\..\ACC_file.c
```

Comment:
Consider to remove compilation files from repository and add proper instruciton how to compile. Justo to keep repository clean